index.html

```diff
@@ -50,6 +50,264 @@ <h1>I'm Matunda. Glad to see you!</h1>
         </div>
      </div>
     </section>
+    <!--Works Section-->
+    <section>
+      <div class="grid-container">
+        <div class="grid-card small-card">
+          <div class="snapshot">
+            <img src="./images/Tonic.svg">
+          </div>
+          <div class="project-details">
+            <div class="project-name"><h2>Tonic</h2></div>
+          
+            <div class="project-more-details">
+              <div class="client">
+                <p>CANOPY</p>
+                              
+              </div>
+              <img class="counter-img" src="./images/Counter.svg"> 
+              <div class="role">
+                <p>Back End Dev</p> 
+                
+              </div>
+              <img class="counter-img" src="./images/Counter.svg"> 
+              <div class="year">
+                <p>2015</p>
+              </div>
+            </div>
+
+          </div>
+          <div class="summary">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          </div>
+          
+            <ul class="tags">
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>Javascript</li>
+            </ul>
+           
+          <div class="action">
+            <div class="see-project">
+              <a href="#"><p>See Project</p></a>
+            </div>
+          </div>
```

Comment:
[OPTIONAL]
- [ ] Same too :point_up_2: , remove the extra `<div class="action">`